src/Components/MessageMetadataInput.tsx

```diff
@@ -0,0 +1,122 @@
+import type { Message } from '../types/proceedings';
+import { TransportMediumChooser } from './Generator/TransportMediumChooser';
+import { Radio } from './Radio';
+import { Text } from 'preact-i18n';
+import t from '../Utility/i18n';
+
+type MessageMetadataInputProps = {
+    message: Omit<Message, 'id'>;
+    includeContent?: boolean;
+    idSuffix?: string;
+    onChange: (message: Omit<Message, 'id'>) => void;
+};
+
+export const MessageMetadataInput = (props: MessageMetadataInputProps) => {
+    return (
+        <div className="message-metadata-input">
+            <div className="form-group">
+                <label htmlFor={`message-metatdata-input-date${props.idSuffix ? '-' + props.idSuffix : ''}`}>
+                    <Text id="date" />
+                </label>
+                <input
+                    id={`message-metatdata-input-date${props.idSuffix ? '-' + props.idSuffix : ''}`}
+                    type="date"
+                    className="form-element"
+                    value={props.message.date.toISOString().split('T')[0]}
```

Comment:
That's what we typically use.

```suggestion
                    value={props.message.date.toISOString().substring(0, 10)}
```